### DISCUSSION TITLE
Bugging Teleportation?

### DISCUSSION THREAD

**Discussion by GoldenGamerLP** [Minestom/Minestom#711] [Help]
Hello, i wanted to ask if there is a fix, or anyone who can explain why the players gets resetted twice sometimes, and sometimes teleported outside the border? (see the video)

- Video:
- https://user-images.githubusercontent.com/65727384/155209314-18b2660a-bdc3-4e40-8758-1285ff32ee7d.mp4

Code:
- Method to reset player:
![image](https://user-images.githubusercontent.com/65727384/155209441-6a2c69fa-d2e8-4ec9-9e00-38b03737c4dd.png)

- ArenaPlayer class:
![image](https://user-images.githubusercontent.com/65727384/155209525-ab90a18d-698a-4523-9fdc-eb9b65e7b7e9.png)

- Scheduler where player gets updated/ resetted:
![image](https://user-images.githubusercontent.com/65727384/155209663-0d21e76b-30cf-404d-8fcd-d7a2832d2b63.png)

- Method to reset player:
![image](https://user-images.githubusercontent.com/65727384/155209774-839d2784-4f01-487b-95db-d3ef306f3c8c.png)

Thank you in advance!

**Comment by GoldenGamerLP** [comment]
Push

**Comment by Moulberry** [comment]
At what point in the video do you get teleported out of the border

**Comment by GoldenGamerLP** [comment]
This one:
![image](https://user-images.githubusercontent.com/65727384/155983285-9c8cadaa-29fb-4052-bd2a-0791854e6b4e.png)

Side note: Sometimes the player stays for some ticks in the air and just dont move at all

**Reply by Moulberry** [reply]
Have you tried debugging the problem?

**Reply by Moulberry** [reply]
If so, please provide that information. A video and your code isn't enough to help find your issue

**Reply by GoldenGamerLP** [reply]
What should I debug then?
